### PR TITLE
feat(utxo-ord): add support for additional inputs

### PR DESCRIPTION
**Pull request by OttoAllmendinger**
A single input might be too small to allow building an inscription
passing transaction

Issue: BG-68315

### CODE REVIEW

**Review by davidkaplanbitgo — DISMISSED** [review]

File: modules/utxo-ord/src/psbt.ts [hunk 1]
```diff
@@ -115,8 +120,11 @@ export function findOutputLayoutForWalletUnspents(
     maxInscriptionOutput = BigInt(20_000),
   } = constraints;
 
-  const [input] = inputs;
-  const inscriptionOutput = new OrdOutput(input.value, [toSatRange(satPoint)]);
+  // Join all the inputs into a single inscriptionOutput.
+  // For the purposes of finding a layout there is no difference.
+  const inscriptionOutput = OrdOutput.joinAll(
+    inputs.map((i) => new OrdOutput(i.value, i === inputs[0] ? [toSatRange(satPoint)] : []))
```
Comment (brandonblack) [inline modules/utxo-ord/src/psbt.ts:126]: Hmm, this assumes that the order of inputs is set by the caller. kinda seems like we'd want to do something more like having the inscribed input in one param and then the other available inputs in a separate param so they can be added where needed to make reasonable sized fee and change outs.

**Review by brandonblack — CHANGES_REQUESTED** [review]

File: modules/utxo-ord/src/psbt.ts (hunk 1 above)
Comment (OttoAllmendinger) [reply modules/utxo-ord/src/psbt.ts:136]: fixed

Comment (OttoAllmendinger) [reply modules/utxo-ord/src/psbt.ts:147]: * new argument `supplementalUnspents: WalletUnspent[]`
* new argument `minimizeInputs`

**Review by davidkaplanbitgo — APPROVED** [review]

File: modules/utxo-ord/src/psbt.ts [hunk 2]
```diff
@@ -135,19 +159,84 @@ export function findOutputLayoutForWalletUnspents(
...
+  }: {
+    supplementaryUnspents?: WalletUnspent[];
+    minimizeInputs?: boolean;
+  } = {}
 ): bitgo.UtxoPsbt {
-  const layout = findOutputLayoutForWalletUnspents(unspents, satPoint, outputs, constraints);
-  if (!layout) {
+  // support for legacy call style
+  if (Array.isArray(unspent)) {
+    if (unspent.length !== 1) {
```
Comment (mmcshinsky-bitgo) [inline modules/utxo-ord/src/psbt.ts:223]: Can we ignore the latter unspents and just take the first one without throwing an error?

**Review by brandonblack — APPROVED** [review]

**Pull request merged**